Assets/BowAndArrowImproved/Scripts/Leaderboard/Leaderboard.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+public class Leaderboard : MonoBehaviour
+{
+    private const string JsonLeaderBoardFileName = "JsonLeaderboardHolder.json";
```

Comment:
change the name to snake case all caps so we know it's a constant and not a variable, like so: 
```
JSON_LEADERBOARD_FILE_NAME = ...
```